### ISSUE TITLE
Where appropriate add "Rails specific" lookbook examples.

### ISSUE THREAD

**Issue by dan-gamble** [baoagency/polaris_view_components#103]
Currently I've been straight copying 1:1 the Polaris examples. It would be good to show some Rails-esque ways of using the library. Mostly around the form components.

**Comment by kirillplatonov** [comment]
Yes. We could add custom FormBuilder and wrap common rails helpers (eg `form.text_field`, `form.select`) with components automatically.
